src/AstTranslator.cpp

```diff
@@ -833,14 +833,22 @@ std::unique_ptr<RamStatement> AstTranslator::ClauseTranslator::translateClause(
                 }
             }
 
+            // check whether all arguments are unnamed variables
+            bool isAllArgsUnnamed = true;
+            for (size_t pos = 0; pos < atom->argSize(); ++pos) {
```

Comment:
Can you use something like
```suggestion
            for (auto* argument: atom-getArguments()) {
```
(than use `argument` instead of `atom-.>getArgument(pos)`) instead? If so, use that.

Reply:
Done.